app/consolidator/FormConsolidatorActor.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package consolidator
+
+import akka.actor.{ Actor, Props }
+import consolidator.scheduler.JobParam
+import org.slf4j.{ Logger, LoggerFactory }
+
```

Comment:
FormConsolidatorActor receives the job param details through the quartz scheduler and does the consolidation work aka reading data from mongo db, generating CSV and posting to DMS